### ISSUE TITLE
Password detail is lost when switching to another app.

### ISSUE THREAD

**Issue by johnmee** [mssun/passforios#555]
I typically need to copy and paste the username/email, then come back and copy/paste the password, and sometimes come back for a totp code.  Each time I need to search for and select the desired password entry.

It did not used to do this; when I came back it would still have the chosen password entry on screen.  I have upgraded my phone from iOS 14 on an old button id phone 6s, to a face id Xs with ios15, but I have no idea if that has anything to do with this problem.

Do others experience the password selection screen reverting to the index on every return from another app?  Is the change in behaviour due to faceid, or iOS, or the app?  And can we get it to remember where we were again?

**Comment by agostonbarna** [comment]
Yep, I have the same issue with the latest pass version (0.12.0) and iOS version (15.4.1) on an iPhone 13, and I also use pass with face ID.

**Comment by samrjack** [comment]
I've been using this app for quite a while and this here is my only complaint. I store a lot of things that can't be pre-populated and needing to re-find the entry every time is such a pain!
